bucket4j/src/main/java/com/linecorp/armeria/server/throttling/bucket4j/TokenBucket.java

```diff
@@ -78,7 +78,7 @@ public static TokenBucket of(long limit, Duration period) {
      * Defines throttling configuration comprised of zero or more bandwidth limits in accordance to
      * token-bucket algorithm.
      *
-     * <h3>Multiple bandwidths:</h3>
+     * <h2>Multiple bandwidths:</h2>
```

Comment:
is JDK 15 strict about h3 without h2?

Reply:
Yes, the build fails if headings are used without the order.